spec/javascript/packages/document-capture/components/documents-step-spec.jsx

```diff
@@ -82,4 +85,39 @@ describe('document-capture/components/documents-step', () => {
 
     expect(queryByText(notExpectedText)).to.not.exist();
   });
+
+  context('not ready section', () => {
+    it('is rendered when enabled', () => {
+      const App = composeComponents(
+        [
+          UIConfigContext.Provider,
+          {
+            value: {
+              notReadySectionEnabled: true,
+            },
+          },
+        ],
+        [DocumentsStep],
+      );
+      const { getByRole } = render(<App />);
+      expect(getByRole('heading', { name: 'doc_auth.not_ready.header', level: 2 })).to.be.ok();
+      const button = getByRole('button', { name: 'doc_auth.not_ready.button_nosp' });
+      expect(button).to.be.ok();
+    });
+    it('is rendered when disabled', () => {
```

Comment:
Just a small suggestion for test clarity.

```suggestion
    it('is not rendered when disabled', () => {
```

Reply:
@night-jellyfish, done!